setup.py

```diff
@@ -1,15 +1,40 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(name='ptitprince',
       version='0.2.5',
       description='A Python implementation of Rainclouds, originally on R, ggplot2. Written on top of seaborn.',
-      url='http://github.com/pog87/PtitPrince',
+      long_description=open('README.md').read(),
+      long_description_content_type='text/markdown',      
```

Comment:
adding the "long description" will make it so that the page on pypi (https://pypi.org/project/ptitprince/) is no longer blank:

![Uploading image.png…]()